### PR TITLE
Replaces old array sprites

### PR DESCRIPTION
**Pull request by Sparkezel**
## About The Pull Request
Replaces array sprites with new ones!

## Why It's Good For The Game
They look much nicer compared to the old ones, follow 3:4 and make new SM look less out of place 
image

## Changelog
:cl: SparkezelPL
imageadd: Adds new radiation collector array sprites. If an array is turned on but has no tanks, it now displays an error animation.
spellcheck: Removes "Hawking Radiation" in arrays with just radiation. It was nice to know you singulo...
/:cl:

### CODE REVIEW

**Comment by CRITAWAKETS** [comment]
massive perspective mismatch between the top and bottom, top is WAYYY too shiny, bottom needs details and loooks worse than our current sprite.

**Comment by Tlaltecuhtli** [comment]
make it animated

**Comment by Sparkezel** [comment]
Here's the animation

https://user-images.githubusercontent.com/53384660/109393584-e901f280-7922-11eb-8194-543253364485.mp4

**Comment by Tlaltecuhtli** [comment]
the bubblegum thing could have some fluid animations

**Comment by itseasytosee** [comment]
soul...

**Comment by MrDoomBringer** [comment]
make that white 'shine' line across the top a softer shade of pink please
other than looks good
would be cool if it had directionals tbh

**Comment by Sparkezel** [comment]
> make that white 'shine' line across the top a softer shade of pink please
> other than looks good
> would be cool if it had directionals tbh

I kinda saved my work file as ping.... pain

**Comment by Iatots** [comment]
Candy dispenser for candy SM.
The bottom is so plain and smooth, it looks like an afterthought.

**Comment by Sparkezel** [comment]
old arrays looked even worse, don't lie to me
image

**Comment by Sparkezel** [comment]
Uploading betterer.mp4…

**Comment by Sparkezel** [comment]
https://user-images.githubusercontent.com/53384660/109425928-131ee780-79eb-11eb-8983-350cfd054927.mp4

**Review by Twaticus — APPROVED** [review]

**Comment by Mothblocks** [comment]
i think it'd be cool if they overshot a little and came back down but thats just me

File: code/modules/power/singularity/collector.dm [hunk 1]
```diff
@@ -184,7 +184,9 @@
 		. += "ptank"
 	if(machine_stat & (NOPOWER|BROKEN))
 		return
-	if(active)
+	if(active && !loaded_tank)
+		. += "error"
+	if(active && loaded_tank)
 		. += "on"
```
Comment (Mothblocks) [inline code/modules/power/singularity/collector.dm:190]:
```suggestion
	if(active)
		. += loaded_tank ? "on" : "error"
```

**Comment by Mothblocks** [comment]
unmark your pr from draft if you are ready

**Pull request merged**